### PR TITLE
Add datasets for star galaxy classifiers

### PR DESCRIPTION
**Pull request by sr525**

### CODE REVIEW

File: policy/datasets.yaml [hunk 1]
```diff
@@ -415,6 +415,20 @@ ref_cat:
...
+    storage: PickleStorage
+    table: ignored
+    template: algorithmData/starGalaxyClassifier/%(version)s/clfIsotonicColour.pickle
+starGalaxy_morphOnlyClassifier:
+    description: "The trained star galaxy separation classfier - uses morphology information only"
+    persistable: ignored
+    python: sklearn.calibration.CalibratedClassifierCV
+    storage: PickleStorage
+    table: ignored
+    template: algorithmData/starGalaxyClassifier/%(version)s/clfIsotonicMorph.pickle
```
Comment (mrawls) [inline policy/datasets.yaml]: These two classifiers appear to be the only use of a `version` keyword in a file path template. I'm wondering if it's strictly necessary or if there is a better way to handle versioning than via file paths?

Comment (RobertLuptonTheGood) [reply policy/datasets.yaml]: Good catch.  It'd be worth talking to @TallJimbo about Gen3 butler approaches to this.

Comment (TallJimbo) [reply policy/datasets.yaml]: FWIW, I'm okay with this approach, at least right now (I'd discussed it with @sr525 previously), and it's not obvious the answer will change in Gen3, though the current nomenclature would be to use "label" rather than "version" as the data ID key.  There will be at least two other possible alternative approaches in Gen3 (different collections and different DatasetTypes), but at least right now it feels like this will remain the best approach there.

Comment (mrawls) [reply policy/datasets.yaml]: Fair enough, thanks for this context. I prefer `label`, so if we can please switch to that instead of `version`, I'll consider this reviewed.
